src/Symfony/Component/DependencyInjection/Attribute/Autoconfigure.php

```diff
@@ -21,7 +21,7 @@ class Autoconfigure
 {
     /**
      * @param array<array-key, array<array-key, mixed>>|string[]|null $tags         The tags to add to the service
-     * @param array<string, array<array-key, mixed>>|null             $calls        The calls to be made when instantiating the service
+     * @param array<array-key, array<array-key, mixed>>|null          $calls        The calls to be made when instantiating the service
```

Comment:
let's simplify then?
```suggestion
     * @param array<array<mixed>>|null          $calls        The calls to be made when instantiating the service
```

Reply:
Yes, shure.